test/Feature/RawOutputTypesSpec.hs

```diff
@@ -0,0 +1,24 @@
+module Feature.RawOutputTypesSpec where
+
+import Network.Wai (Application)
+
+import Network.HTTP.Types
+import Test.Hspec
+import Test.Hspec.Wai
+
+import Protolude
+import SpecHelper (acceptHdrs)
+
+spec :: SpecWith Application
+spec = describe "When raw-output-media-types config variable is missing or left empty" $ do
+  let firefoxAcceptHdrs = acceptHdrs "text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8"
+      chromeAcceptHdrs = acceptHdrs "text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3"
```

Comment:
Really nice! This is what I also had in mind.

One thing though, I think it'd be better to test an endpoint such as `/items?id=lte.3` and make sure we get the JSON payload. Root `/` follows some different rules.

To be extra sure we could also test a GET on rpc, an example `/rpc/get_projects_below?id=3`.

Reply:
Ok. I'll be "afk" till the 21st but will try to fix them asap.

Thanks for the feedback :)